### DISCUSSION TITLE
Auto Instrumentation and JaegerExporter keeps getting [WARNING] io exception and error connecting at port 4317

### DISCUSSION THREAD

**Discussion by stevensim226** [open-telemetry/opentelemetry-java-instrumentation#2760] [Q&A] [answered]
Hello,
I'm trying to setup OpenTelemetry w/ the javaagent automatic instrumentation on my code base using Jaeger as the export target.

I followed the steps in this documentation https://opentelemetry.io/docs/java/automatic_instrumentation/#jaeger-exporter and by that I added 3 system properties values which are
```
-Dotel.exporter=jaeger
-Dotel.exporter.jaeger.endpoint=localhost:14250
-Dotel.exporter.jaeger.service.name=sample-service
```
and also the automatic instrumentation `.jar` file as javaagent
4 of those settings has been set correctly 

But whenever I start the app it keeps giving me this warning
`[opentelemetry.auto.trace 2021-04-09 00:37:40:121 +0700] [grpc-default-executor-1] WARN io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter - Failed to export spans. Error message: UNAVAILABLE: io exception`

and also at times it will also cause a Java exception which says
`Connection refused: localhost/0:0:0:0:0:0:0:1:4317`
despite Jaeger settings not needing port `4317` according to Jaeger's docs for OpenTelemetry https://www.jaegertracing.io/docs/1.21/opentelemetry/

Also when I test to access a function with a `@WithSpan` annotation, it also doesn't export the span into Jaeger correctly

Is there something that I forgot to specify? after what I found port `4317` is for `OTLP/gRPC` but then I'm using `JaegerExporter` which should be using `14250`

**Comment by trask** [comment, marked as answer]
hi @stevensim226! sorry, we need to update those docs. in the meantime, checkout https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#jaeger-exporter for the latest settings

**Reply by stevensim226** [reply]
Hi @trask 
Thanks for the information, now it's working as expected
